docs/source/spec/core.rst

```diff
@@ -2378,30 +2361,23 @@ Consider the following metadata definition:
 
 .. code-block:: smithy
 
-    namespace smithy.example
-
     metadata foo = {
-        MyString: MyString,
+        String: String,
     }
 
-    string MyString
+    string String
```

Comment:
Bit of a confusing example - is the metadata referencing `smithy.api#String` like described or this `string String`? If the former, can the `string String` be removed? If `string String` should stay, this needs a namespace.

Reply:
Good catch. Removes the `string String`